garrysmod/gamemodes/terrortown/gamemode/traitor_state.lua

```diff
@@ -179,6 +179,7 @@ local function force_spectate(ply, cmd, arg)
       end
    end
 end
-concommand.Add("ttt_spectate", force_spectate)

```

Comment:
Don't delete the concommand.
This would break things.
https://github.com/garrynewman/garrysmod/blob/master/garrysmod/gamemodes/terrortown/gamemode/cl_help.lua#L162-#L168
Also there could be addons which are using this addon.